collada/schema.py

```diff
@@ -14,22 +14,36 @@
 """This module contains helper classes and functions for working
 with the COLLADA 1.4.1 schema."""
 
+import os
 import lxml
 import lxml.etree
 from collada.util import bytes, BytesIO
 
-from pkg_resources import resource_string
+# the absolute directory of this file
+_cwd = os.path.abspath(os.path.expanduser(os.path.dirname(__file__)))
 
-# get a copy of the XML schema
-# resource_string returns bytes so decode into string
-COLLADA_SCHEMA_1_4_1 = str(resource_string(
-    'collada',
-    'resources/schema-1.4.1.xml').decode('utf-8'))
+def resource_string(file_name: str) -> str:
```

Comment:
Could you fix the failed formatting check? Needs two blank lines before top level functions

Reply:
Done! I also as an option made an alternative PR #149 which switches to [ruff](https://github.com/astral-sh/ruff). I've found it to be pretty nice and it has a lot of ecosystem momentum, but linters and formatters are pretty subjective and both ruff/flake8 seem fine to me.